### DISCUSSION TITLE
Add pictures to show how the widget looks in the README.md file

### DISCUSSION THREAD

**Discussion by HazardCodeBolt** [nixrajput/flutter_carousel_widget#10] [Ideas]
I have seen about this dependency in pub.dev, but in the main page there is no image that illustrates how will the widget look after it is used. I think it is better for you to include an image or a GIF that shows the different ways to use this widget.

**Comment by nixrajput** [comment]
Hi @HazardCodeBolt,

Thank you for using my package.

I added the screenshots in previous versions.
But unfortunately, `githubusercontent` domain is not working, so I have decided to remove it.
I will add the screenshots in the next update.
Alternatively, you can visit the [demo](https://nixrajput.github.io/flutter_carousel_widget) web app for illustration.

Thanks

**Reply by HazardCodeBolt** [reply]
Well, 
Your package carousels are well designed and smooth, and that is what I liked about it. 
I found out that there is a demo in the documentation after you said it. I was a little in a hurry just to decide what package to use. 
Thank you for the response.
